_errata/errata-print02-ch02-lst03-superfluous-punctuation

```diff
@@ -0,0 +1,30 @@
+---
+printing: second
```

Comment:
No need for this — the error is present in the first printing as well as far as I can tell. Same with the file name :+1: